elasticsearch_app/search.py

```diff
@@ -148,6 +149,11 @@ def prepare_title(resource):
     return resource.title
 
 
+# TODO: Should we display results this way?
```

Comment:
Not currently used, but the point of this was to allow search suggestions to be displayed as such:
LayerTitle (Layer)
DocumentTitle (Document)
MapTitle (Map)

Where the title is what is being autocompleted, but the user is given information on what type of data the title is for in parentheses.